bin/bootstrax

```diff
@@ -490,22 +490,27 @@ def send_heartbeat(update_fields=None):
     set_state(None, update_fields=update_fields)
 
 
-def log_warning(message, priority='warning'):
+def log_warning(message, priority='warning', run_id=None):
     """Report a warning to the terminal (using the logging module)
     and the DAQ log DB.
+    :param message: insert string into log_coll
     :param priority: severity of warning. Can be:
         info: 1,
         warning: 2,
         <any other valid python logging level, e.g. error or fatal>: 3
+    :param run_id: optional run id.
     """
     ping_dbs()
     if not args.production:
         return
     getattr(log, priority)(message)
-    log_coll.insert_one({
+    warning_message = {
         'message': message,
         'user': f'bootstrax_{hostname}',
-        'priority': dict(warning=2, info=1).get(priority, 3)})
+        'priority': dict(warning=2, info=1).get(priority, 3)}
+    if run_id:
+        warning_message.update({'run_id': run_id})
```

Comment:
Redax uses `runid` as the key (see [here](https://github.com/coderdj/redax/blob/master/MongoLog.cc#L141)). Also, I think pep8 recommends the explicit `if run_id is not None:`, but that's stylistic. Also, is it worth adding the full hierarchy of error messages from `debug` up to `fatal`? It seems like just two could be too limiting.